### ISSUE TITLE
top page nav disappears on mobile

### ISSUE THREAD

**Issue by imatwawana** [getsentry/sentry-docs#4593]
### Core or SDK?

Core Sentry product

### Which part? Which one?

All

### Description

The top nav (see image below) disappears when you navigate away from the landing page of Docs on mobile.

On landing page: 
![Image from iOS (2)](https://user-images.githubusercontent.com/76437239/149559558-efeb0c2c-6c09-4b06-a423-c3cf8a11e37b.png)

On other pages:
![image](https://user-images.githubusercontent.com/76437239/149559612-93d48281-f765-4fe1-9e44-5dbbdd0d71f8.png)

### Suggested Solution

One option could be to add them the end of the list in the table of contents

**Comment by Neeleshrj** [comment]
Hi, can I work on this issue?

**Comment by imatwawana** [comment]
> Hi, can I work on this issue?

Yep, go for it. I see you put up a PR and I've added one of our devs and myself as reviewers.
